### DISCUSSION TITLE
plotmafSummary

### DISCUSSION THREAD

**Discussion by Gioylako** [PoisonAlien/maftools#791] [Q&A]
Hi I am new in this and I am wondering why in plotmafSummary the percentages are not in order from bigger to smaller ? I found in different files this error where 17 % comes before 24 % while the bars are same or even the 17 % is bigger than 24%

Best 

Georgia

**Comment by PoisonAlien** [comment]
Hello,

Please see issues #339 and #620 for details.
Also this seems to be most frequently asked question. Maybe I should change the plot :)
